requirements/edx/base.txt

```diff
@@ -38,13 +38,13 @@ git+https://github.com/edx/django-rest-framework.git@3c72cb5ee5baebc432894737119
 django==1.8.14
 djangorestframework-jwt==1.8.0
 djangorestframework-oauth==1.1.0
-edx-ccx-keys==0.1.2
+edx-ccx-keys==0.2.1
 edx-drf-extensions==0.5.1
 edx-lint==0.4.3
 edx-django-oauth2-provider==1.1.1
 edx-django-sites-extensions==2.1.1
 edx-oauth2-provider==1.1.3
-edx-opaque-keys==0.3.3
+edx-opaque-keys==0.3.4
```

Comment:
The minimum version listed in `common/lib/xmodule/setup.py` is still 0.2.1; is that still a sufficient version?  Also, https://github.com/edx/opaque-keys/blob/master/CHANGELOG.rst doesn't seem to have been updated for the 0.3.4 release.